src/App.js

```diff
@@ -32,6 +33,7 @@ function App() {
               <Route path="/reward/coupons" element={<Coupons />} />
               <Route path="/user-home/join-challenge" element={<JoinChallange />} />
               <Route path="/user-home/join-challange-success" element={<JoinChallangeSuccess />} />
+              <Route path="/recycle/chat-with-recycle-companies" element={<Chat />} />
```

Comment:
If we are really aiming for full REST, the path should be `recycle/chat/:recycleCompanyId`, but I think we're good here. Nice work guys!

Reply:
@ziyangcS2023 @YinzhuSU Just FYI ⬆️ I'm resolving it rn.